CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
  - display `GooseStats` with fmt::Display (ie `print!("{}", goose_stats);`)
  - make it possible to pass a closure to GooseTask::new
  - fix display of `GooseError` and `GooseTaskError`
+ - default to resetting statistics, disable with `--no-reset-stats`, display spawning statistcs before resetting
```

Comment:
nit - typo

Reply:
I've just now added a spell-checker to my IDE to avoid needing this sort of nit in the future. Thanks.